### PR TITLE
Add Injective chain

### PR DESCRIPTION
**Pull request by 4coffeeihadbreakfast**
Mostly these are the values suggested by crescent.network when connecting. (not totally sure they are correct but did the yarn checks, so is at least valid)

Also, was getting eslint errors, so added the ignore line

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ const main = async () => {
     for (const imageFile of imageFiles) {
       checkImageSize(`images/${chainIdentifier}/${imageFile}`);
     }
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (HeesungB) [inline src/index.ts:94]: You should delete this commnet

File: src/schedule.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ const main = async () => {
   let errorMessages: (
     | {
         file: string;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (HeesungB) [inline src/schedule.ts:10]: You should delete this commnet

**Comment by HeesungB** [comment]
@4coffeeihadbreakfast 
Thank you for contribution.

EVM Chain should add `eth-address-gen, eth-key-sign` features.
Please add `eth-address-gen, eth-key-sign` in features

File: src/index.ts (hunk 1 above)
Comment (4coffeeihadbreakfast) [reply src/index.ts:94]: https://typescript-eslint.io/rules/no-explicit-any/

File: src/schedule.ts (hunk 2 above)
Comment (4coffeeihadbreakfast) [reply src/schedule.ts:10]: https://typescript-eslint.io/rules/no-explicit-any/

**Comment by HeesungB** [comment]
You should remove `coinType` field.

And Keplr will add Injective as native chain.

**Pull request merged**